compiler/optimizer/OSRDefAnalysis.cpp

```diff
@@ -1474,26 +1754,37 @@ void TR_OSRLiveRangeAnalysis::buildOSRLiveRangeInfo(TR::Node *node, TR_BitVector
    *_workBitVector -= *liveVars;
 
    TR_BitVector *deadSymRefs = NULL;
+   TR_BitVector deadSymRefsTemp(0, comp()->trMemory()->currentStackRegion());
```

Comment:
Other temporary bitvectors in this optimization are allocated once and cleared, such as `_workBitVector`. I'm not sure if this is actually worthwhile for compile time, just an observation.

Reply:
Thanks for pointing that out! Changed the deadSymRefsTemp to work the same way as _workBitVector to save compilation time.